recipes/zitadel-client/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+    version: 4.1.0b6
+    build_number: 0
```

Comment:
Single use variables obscures the recipe. Please more the value to the entry below and remove this one.